scripts/safe_run.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
`safe_run.sh` is sort of a weird name that doesn't immediately tell you what this file does

But tbh I don't know a better name... maybe `ci_runner.sh` is better. At least it tells you that it's only relevant for CI.